readme.md

```diff
@@ -33,6 +33,12 @@ $ de --help
 
 It's `de` and not `del`, since `del` is already a builtin command on Windows.
 
+If preferred, a simple alias could fix this of course.
+
+```
+	alias del=de
```

Comment:
thought it would be helpful ;)

On Fri, Jan 22, 2016 at 11:17 AM Michael Mayer <notifications@github.com>
wrote:

> In readme.md
> <https://github.com/sindresorhus/del-cli/pull/5#discussion_r50521093>:
>
> > @@ -33,6 +33,12 @@ $ de --help
> >
> >  It's `de` and not `del`, since `del` is already a builtin command on Windows.
> >
> > +If preferred, a simple alias could fix this of course.
> > +
> > +```
> > +	alias del=de
>
> Also nitpicking, but that depends on your shell, e.g.:
>
> # tcsh and cshalias del de
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sindresorhus/del-cli/pull/5/files#r50521093>.
>
-- 
--

*Dimitrie Hoekstra, UI & Interaction Designer*
dhesign.com  | dimitriehoekstra@gmail.com | LinkedIn
<https://www.linkedin.com/in/dimitrieh> | Twitter
<https://twitter.com/strages33/>